src/main/java/frc/robot/motion_profiling/Path.java

```diff
@@ -0,0 +1,74 @@
+package frc.robot.motion_profiling;
+
+import java.util.Arrays;
+
+import edu.wpi.first.wpilibj.trajectory.Trajectory;
+import edu.wpi.first.wpilibj.trajectory.TrajectoryConfig;
+import edu.wpi.first.wpilibj.trajectory.TrajectoryGenerator;
+import edu.wpi.first.wpilibj.trajectory.constraint.CentripetalAccelerationConstraint;
+import frc.robot.Robot;
+
+import static frc.robot.Robot.robotConstants;
+
+/**
+ * Class for creating a new path for motion profiling path following.  
+ */
+public class Path {
+    private Trajectory trajectory;
+    private final TrajectoryConfig config;
+    private boolean reversed;
+    private double startVelocity;
+    private double endVelocity;
+    
+    public Path(Waypoint... waypoints) {
+        this(false, robotConstants.motionProfilingConstants.DEFAULT_START_PATH_VELOCITY, 
+                    robotConstants.motionProfilingConstants.DEFAULT_END_PATH_VELOCITY, waypoints);
+    }
+
+    public Path(boolean reversed, Waypoint... waypoints) {
+        this(reversed, robotConstants.motionProfilingConstants.DEFAULT_START_PATH_VELOCITY, 
+                    robotConstants.motionProfilingConstants.DEFAULT_END_PATH_VELOCITY, waypoints);
+    }
+
+    /**
+     * Creates new path with reversed, start vel & end vel params. 
+     * @param reversed whether the path is reversed
+     * @param startVelocity the velocity at the start of the path 
+     * @param endVelocity the velocity at the end of the path 
+     * @param waypoints the path waypoints 
+     */
+    public Path(boolean reversed, double startVelocity, double endVelocity, Waypoint... waypoints) {
```

Comment:
When we would want to change the start velocity?
Maybe It's a good idea to add a constructor that only changes end velocity?